Hunting Queries/AuditLogs/AccountAddedtoPrivilegedPIMGroup

```diff
@@ -0,0 +1,29 @@
+name: AccountAddedtoPrivilegedPIMGroup
+description: |
+  'Identifies accounts that have been added to a PIM managed privileged group'
+requiredDataConnectors:
+  - connectorId: Azure Active Directory
+    dataTypes:
+      - AuditLogs
+tactics:
+  - Persistence
+  - PrivilegeEscalation
+relevantTechniques:
+  - T1098
+  - T1548
+query: |
+AuditLogs
+| where TimeGenerated > ago(1d)
+| where ActivityDisplayName =~ "Add eligible member to role in PIM requested (timebound)"
+| where AADOperationType =~ "CreateRequestEligibleRole"
+| where TargetResources contains "-PRIV"
+or TargetResources contains "Administrator"
+or TargetResources contains "Security"
```

Comment:
please change to use has_any instead of contains .
`| where TargetResources has_any ("Administrator", "-PRIV" , "Security")`